packages/grafana-ui/src/components/CustomScrollbar/_CustomScrollbar.scss

```diff
@@ -13,15 +13,15 @@
 
   .track-vertical {
     border-radius: 3px;
-    width: 6px !important;
+    width: 8px !important;
```

Comment:
this is unrelated to main PR but good change, the custom scrollbars are to narrow